docs/Programming Framework.md

```diff
@@ -76,6 +76,10 @@ IPF can be edited using INAV Configurator user interface, of via CLI
 | 40            | MOD           | Divide `Operand A` by `Operand B` and returns the remainder |
 | 41			| LOITER_RADIUS_OVERRIDE | Sets the loiter radius to `Operand A` [`0` : `100000`] in cm. If the value is lower than the loiter radius set in the **Advanced Tuning**, that will be used. |
 | 42            | SET_PROFILE   | Sets the active config profile (PIDFF/Rates/Filters/etc) to `Operand A`. `Operand A` must be a valid profile number, currently from 1 to 3. If not, the profile will not change |
+| 43			| CONSTRAIN_MIN		| Ensures that `Operand A` cannot be less than `Operand B` |
```

Comment:
I think wording can be misleading. And if not counting equal, CONSTRAIN_MIN is the same as MAX.

Can we maybe have only MIN and MAX operators?

Reply:
Yes, you're right. MAX works as CONSTRAIN MIN, and vice versa. So no need for 2 sets of operators. Removed the CONSTRAIN operations, as MIN and MAX is simper to understand.